src/hooks/useAxios.jsx

```diff
@@ -1,6 +1,7 @@
 import { useState } from "react";
 import axios from "axios";
 
+// NOTE: This is not being used anywhere
```

Comment:
Is this comment still valid? If so, you can remove this page, else delete the comment

Reply:
I might try and use it later maybe, so kept it here for future reference